gratipay/security/authentication.py

```diff
@@ -82,9 +81,7 @@ def add_auth_to_response(response, request=None, user=ANON):
     if request is None:
         return  # early parsing must've failed
     if request.line.uri.startswith('/assets/'):
-        return  # assets never get auth headers and have their own caching done elsewhere
-
-    response.headers['Expires'] = BEGINNING_OF_EPOCH # don't cache
```

Comment:
> we don't need to use `Expires` anymore

... for authed resources, since authed resources are a strict subset of dynamic resources?